content/practices/template/index.md

```diff
@@ -0,0 +1,78 @@
+---
+path: "/methods/template"
+title: "_Template"
+hidden: "true"
```

Comment:
I think what you want here is `draft: true` if you're trying to prevent this page from being crawlable/addressable. Right now, you can still get to this page it looks like.

Reply:
do `draft: true` pages only get hidden at deploy time? I still see them in the `/practices` index locally and I can navigate to them. Regardless I'll remove that page from this batch.

Reply:
I think that's probably because they don't get deleted after the fact so they're left from your previous local deploy.

Reply:
Gotcha. Page removed for now regardless since a new template or archetype is needed.